nuitka/tools/testing/OutputComparison.py

```diff
@@ -202,7 +222,7 @@ def compareOutput(
     diff = difflib.unified_diff(
         makeDiffable(out_cpython, ignore_warnings, ignore_infos, syntax_errors),
         makeDiffable(out_nuitka, ignore_warnings, ignore_infos, syntax_errors),
-        "{program} ({detail})".format(program=os.environ["PYTHON"], detail=kind),
+        "{program} ({detail})".format(program=os.environ["PYTHON"] if "PYTHON" in os.environ else "python", detail=kind),
```

Comment:
Use `os.environ.get` to achieve a default value. However, why is it not set, I think that `setup` function we use for testing, is something you also ought to use and then it's guarantueed to be set.

Reply:
not sure, it wasn't set in my os.environ so this was a quick fix